fdbserver/BlobWorker.actor.cpp

```diff
@@ -605,17 +629,20 @@ ACTOR Future<BlobFileIndex> writeDeltaFile(Reference<BlobWorkerData> bwData,
 	state Optional<BlobGranuleCipherKeysCtx> cipherKeysCtx;
 	state Optional<BlobGranuleCipherKeysMeta> cipherKeysMeta;
 	state Arena arena;
-	// TODO support encryption, figure out proper state stuff
-	/*if (isBlobFileEncryptionSupported()) {
-	    BlobGranuleCipherKeysCtx ciphKeysCtx = wait(getLatestGranuleCipherKeys(bwData, keyRange, &arena));
-	    cipherKeysCtx = ciphKeysCtx;
-	    cipherKeysMeta = BlobGranuleCipherKeysCtx::toCipherKeysMeta(cipherKeysCtx.get());
-	}*/
 
-	Optional<CompressionFilter> compressFilter = getBlobFileCompressFilter();
+	if (isBlobFileEncryptionSupported()) {
+		BlobGranuleCipherKeysCtx ciphKeysCtx = wait(getLatestGranuleCipherKeys(bwData, keyRange, &arena));
+		cipherKeysCtx = std::move(ciphKeysCtx);
```

Comment:
What is this line doing? Doesn't the compiler usually handle moving things intelligently? For non-obvious uses of move it'd be good to add a comment at least so it's not confusing other people who read the code.

Reply:
It might be more readable to just have the next line be this instead?

cipherKeysMeta = BlobGranuleCipherKeysCtx::toCipherKeysMeta(std::move(cipherKeysCtx.get()))

Reply:
Good idea, shall update it